server.js

```diff
@@ -1339,7 +1339,8 @@ app.get('/match-games/:matchid(\\w+)', (req, res) => {
                 html += "<td>" + ipMap.get(item.ip) + "</td>";
                 html += "<td><a href=\"/viewmatch/" + item.sgfhash + "?viewer=wgo\">" + item.sgfhash + "</a></td>";
                 html += "<td>" + item.winnerhash.slice(0,8) + "</td>";
-                html += "<td>" + item.score + "</td><td>" + item.movescount + "</td></tr>\n";
+                html += "<td>" + item.score + "</td><td>" + item.movescount + "</td>";
+				html += "<td><a href=\"/viewmatch/" + item.sgfhash + ".sgf\">sgf</a></td></tr>\n";
```

Comment:
Can you format this with spaces instead of tabs?

Reply:
Sorry for the tabs, I'll adjust my IDE to use space instead of tabs.